### PR TITLE
Add valid parenthesis lab

### PR DESCRIPTION
**Pull request by evanyeyeye**
Simple Python autograder for C code. 

Includes a Dockerfile to build an image with Python.

### CODE REVIEW

File: Cvalidparen/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Instructions:
+# Run: docker build -t autograding_validparen
```
Comment (damianhxy) [inline Cvalidparen/Dockerfile]: Seems to be missing a `.` at the end, i.e. `docker build -t autograding_validparen .`

File: Cvalidparen/Cvalidparen.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (damianhxy) [inline Cvalidparen/Cvalidparen.yml]: For the purposes of this repository, `Cvalidparen.yml` and `Cvalidparen.rb` are not needed and should be removed

**Review by damianhxy — CHANGES_REQUESTED** [review]
Successfully autograded a submission

Left some comments but otherwise LGTM

File: Cvalidparen/Cvalidparen.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+module Cvalidparen
+  include AssessmentBase
+
+  def assessmentInitialize(course)
+    super("Cvalidparen",course)
+    @problems = []
+  end
+
+end
```
Comment (20wildmanj) [inline Cvalidparen/Cvalidparen.rb]: Nit: could you add newlines to the end of files such as here

**Review by michellexliu — COMMENTED** [review]
could you include the tar as well (similar to how there's a hello.tar in https://github.com/autolab/autograders-examples/tree/main/hellotar)

**Review by damianhxy — APPROVED** [review]
successfully built autograder image, installed assessment, and submitted solution

lgtm!

**Pull request merged**